extensions/braze/braze-currents-users-event.schema.json

```diff
@@ -0,0 +1,300 @@
+            },
+            "braze:canvasVariationID": {
+              "title": "Canvas Variation ID",
+              "type": "string",
+              "description": "API ID of the canvas variation this user received."
+            },
+            "braze:canvasVariationName": {
+              "title": "Canvas Variation Name",
+              "type": "string",
+              "description": "Name of the canvas variation this user received."
+            },
+            "braze:canvasStepID": {
+              "title": "Canvas Step ID",
+              "type": "string",
+              "description": "API ID of the canvas step this email belongs to."
+            },
+            "braze:canvasStepName": {
+              "title": "Canvas Step Name",
+              "type": "string",
+              "description": "Name of the canvas step."
+            }
+          }
+        },
+        "braze:messaging": {
+          "title": "Messaging",
+          "type": "object",
+          "properties": {
+            "braze:dispatchID": {
+              "title": "Dispatch ID",
+              "type": "string",
+              "description": "ID of the dispatch this message belongs to."
+            },
+            "braze:sendID": {
+              "title": "Send ID",
+              "type": "string",
+              "description": "Message send ID this message belongs to."
+            },
+            "braze:email": {
+              "title": "Email",
+              "type": "object",
+              "properties": {
+                "braze:sendingIP": {
+                  "title": "Sending IP",
+                  "type": "string",
+                  "description": "IP address from which the email send was made."
+                },
+                "braze:IPPool": {
+                  "title": "IP Pool",
+                  "type": "string",
+                  "description": "IP Pool from which the email send was made."
+                },
+                "braze:ESP": {
+                  "title": "ESP",
+                  "type": "string",
+                  "description": "ESP related to the event (Sparkpost or Sendgrid).",
+                  "enum": [
+                    "SparkPost",
+                    "SendGrid"
+                  ],
+                  "meta:enum": {
+                    "SparkPost": "SparkPost",
+                    "SendGrid": "SendGrid"
+                  }
+                },
+                "braze:fromDomain": {
+                  "title": "From Domain",
+                  "type": "string",
+                  "description": "Sending domain for the email."
+                },
+                "braze:userAgent": {
+                  "title": "User Agent",
+                  "type": "string",
+                  "description": "User agent on which the click occurred."
+                },
+                "braze:machineOpen": {
+                  "title": "Machine Open",
+                  "type": "string",
```

Comment:
Should this be a boolean if it only has true and false ? If yes, let's rename this to "isMachineOpened"

Reply:
It's technically a boolean for now, but I've told by other internal teams that they may want to add additional values other than "true" in the future for additional context, so if that ever happens we won't have the ability to do that if we make this a boolean.